src/ranged.cpp

```diff
@@ -219,13 +223,6 @@ bool player::handle_gun_damage( item &it )
         add_msg_player_or_npc( _( "Your %s misfires with a muffled click!" ),
                                _( "<npcname>'s %s misfires with a muffled click!" ),
                                it.tname() );
-        if( it.damage() < it.max_damage() && one_in( effective_durability ) ) {
```

Comment:
Now that you have removed this block, the comment above is a bit misleading. It talks about "may cause damage", but there is no damage is not applied here.